Formula/git-standup.rb

```diff
@@ -0,0 +1,17 @@
+class GitStandup < Formula
+  desc "Git extension to generate reports for standup meetings"
+  homepage "https://github.com/kamranahmedse/git-standup"
+  url "https://github.com/kamranahmedse/git-standup/archive/1.0.0.tar.gz"
+  sha256 "fd3808fb8414308413248b2f47646f51ce7bf7a6b59beb257fbde2e983b60127"
+  head "https://github.com/kamranahmedse/git-standup.git"
+
+  bottle :unneeded
+
+  def install
+    bin.install "git-standup"
```

Comment:
I noticed that. The reason I didn't use the `make install` target is that it does not allow for `prefix` relocation. I'll raise that topic in the discussion at https://github.com/kamranahmedse/git-standup/issues/9

Should I close this PR in the meantime?